src/WebJobs.Script.WebHost/Diagnostics/FunctionsDiagnosticLogsEventSource.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
+{
+    [EventSource(Guid = "B7AB9421-18D8-4C0A-8389-7CB78944D4A4")]
+    public sealed class FunctionsDiagnosticLogsEventSource : ExtendedEventSource
```

Comment:
Might consider flowing the AzureMonitorDiagnostic name change through the other types/methods

Reply:
I noticed the same thing; I'm doing that now.